Service/pdoShipStorage.php

```diff
@@ -73,14 +73,13 @@ public function saveShip(AbstractShip $ship)
         // maybe return errors if any ecountered
     }
 
-    public function repairShip(AbstractShip $ship)
+    public function repairShip($id)
```

Comment:
Why did you remove passing in the object here? I think that is still needed.

Reply:
Reverted changes.